articles/pivot-terraform-ai-era-design.md

```
@@ -0,0 +1,249 @@
+
+#### PIVOTの状況
+
+PIVOTは1つのプロダクトを1つのチームで開発しており、複数プロダクトを複数チームで開発しているわけではありません。サーバー、フロントエンド、バッチなど、デプロイ単位や開発担当の違いはありますが、これらは「プロダクト」というより「コンポーネント」の分類に近いものです。
+
+チームトポロジーの観点では、複数のプロダクトチームがそれぞれのインフラを管理する構成が理想的なケースもあります。しかし、我々は1チームで全体を見ているため、そのような組織的な境界はありません。
+
+#### 我々の選択: リソース種別ベースで横並び管理
+
+この状況を踏まえ、以下の構成を採用しました：
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)